test/integration/podgc/podgc_test.go

```diff
@@ -40,25 +40,22 @@ import (
 // TestPodGcOrphanedPodsWithFinalizer tests deletion of orphaned pods
 func TestPodGcOrphanedPodsWithFinalizer(t *testing.T) {
 	tests := map[string]struct {
-		enablePodDisruptionConditions bool
 		enableJobPodReplacementPolicy bool
 		phase                         v1.PodPhase
 		wantPhase                     v1.PodPhase
 		wantDisruptionTarget          *v1.PodCondition
 	}{
-		"PodDisruptionConditions enabled": {
-			enablePodDisruptionConditions: true,
-			phase:                         v1.PodPending,
-			wantPhase:                     v1.PodFailed,
+		"pending pod": {
+			phase:     v1.PodPending,
+			wantPhase: v1.PodFailed,
 			wantDisruptionTarget: &v1.PodCondition{
 				Type:    v1.DisruptionTarget,
 				Status:  v1.ConditionTrue,
 				Reason:  "DeletionByPodGC",
 				Message: "PodGC: node no longer exists",
 			},
 		},
-		"PodDisruptionConditions and PodReplacementPolicy enabled": {
-			enablePodDisruptionConditions: true,
+		"pending pod; PodReplacementPolicy enabled": {
```

Comment:
PodReplacementPolicy is irrelevant now, because all its effects on the PodGC are now covered by PodDisruptionConditions=true.